extensions/tokenizers/src/main/python/requirements.txt

```diff
@@ -3,5 +3,5 @@ transformers
 sentence_transformers
 torch
 protobuf==3.20.2
-optimum[exporters,onnxruntime]
```

Comment:
Why remove onnxruntime?

Reply:
This will install onnxruntime cpu version, this will make O4 optimization fail.

Reply:
Changed.